solr/core/src/java/org/apache/solr/core/SolrResourceLoader.java

```diff
@@ -81,11 +82,11 @@
 
 /**
  * @since solr 1.3
- */ 
-public class SolrResourceLoader implements ResourceLoader,Closeable
-{
+ */
+public class SolrResourceLoader implements ResourceLoader, Closeable {
   private static final Logger log = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
 
+  private String name = "";
```

Comment:
Please order this new instance field with the other instance fields, and not with the statics.  Also, shouldn't it be final?  And I wonder if an empty string default is something to improve... it seems to me we could default it to something useful or null.  I prefer to avoid empty strings.